src/test/standardTest.node.ts

```diff
@@ -160,11 +161,23 @@ async function start() {
     console.log('*'.repeat(100));
     console.log('Start Standard tests');
     const platform = computePlatform();
-    const vscodeExecutablePath = await downloadAndUnzipVSCode(channel, platform);
+    const vscodeExecutablePath = await downloadAndUnzipVSCode(channel, platform, undefined, true);
+    console.log(`VS Code executable ${vscodeExecutablePath} exists = ${fs.existsSync(vscodeExecutablePath)}`);
```

Comment:
Left the logs for now, to see why we might be running into 
https://github.com/microsoft/vscode-test/issues/246